bipedalwalker_eng.rst

```diff
@@ -0,0 +1,191 @@
+BipedalWalker
+~~~~~~~~~~~~~~~

```

Comment:
all the files should be moved to `source/env_tutorial`

Reply:
Done